python-avd/pyavd/_eos_cli_config_gen/j2templates/documentation/ethernet-interfaces.j2

```diff
@@ -149,14 +190,29 @@
 
 ##### VLAN Translations
 
-| Interface | From VLAN ID(s) | To VLAN ID | Direction |
-| --------- | --------------- | -----------| --------- |
+| Interface | Direction | From VLAN ID(s) | To VLAN ID | From Inner VLAN ID | To Inner VLAN ID | Network | Dot1q-tunnel |
+| --------- | --------- | --------------- | ---------- | ------------------ | ---------------- | ------- | ------------ |
 {%         for ethernet_interface in ethernet_interfaces | arista.avd.natural_sort('name') %}
-{%             if ethernet_interface.vlan_translations is arista.avd.defined %}
+{%             if ethernet_interface.switchport.vlan_translations is arista.avd.defined %}
+{%                 for vlan_translation in ethernet_interface.switchport.vlan_translations.direction_both | arista.avd.natural_sort('from') %}
+| {{ ethernet_interface.name }} | both | {{ vlan_translation.from }} | {{ vlan_translation.to | arista.avd.default('-') }} | {{ vlan_translation.inner_vlan_from | arista.avd.default('-') }} | - | {{ vlan_translation.network | arista.avd.default('-') }} | {{ vlan_translation.dot1q_tunnel | arista.avd.default('-') }} |
+{%                 endfor %}
+{%                 for vlan_translation in ethernet_interface.switchport.vlan_translations.direction_in | arista.avd.natural_sort('from') %}
+| {{ ethernet_interface.name }} | in | {{ vlan_translation.from }} | {{ vlan_translation.to | arista.avd.default('-') }} | - | {{ vlan_translation.inner_vlan_to | arista.avd.default('-') }} | {{ vlan_translation.network | arista.avd.default('-') }} | {{ vlan_translation.dot1q_tunnel | arista.avd.default('-') }} |
```

Comment:
for in and both direction we have inner vlan from
```
wan1-dc1(config-if-Et1)#switchport vlan translation in 10 ?
  dot1q-tunnel  Dot1q tunnel by retaining the tag
  inner         Inner VLAN ID to map from

wan1-dc1(config-if-Et1)#switchport vlan translation 10 ?
  dot1q-tunnel  Dot1q tunnel by retaining the tag
  inner         Inner VLAN ID to map from
```
but here you are setting it opposite